test/integrations/sources/cordial/data.ts

```diff
@@ -0,0 +1,678 @@
+                cID: '6690fe3655e334xx028xxx',
+              },
+              event: {
+                _id: '669141857b8cxxx1ba0da2xx',
+                cID: '6690fe3655e334xx028xxx',
+                ts: '2024-07-12T14:45:25+00:00',
+                ats: '2024-07-12T14:45:25+0000',
+                d: {
+                  type: 'computer',
+                  device: false,
+                  platform: false,
+                  browser: false,
+                  robot: true,
+                },
+                a: 'browse',
+                tzo: -7,
+                rl: 'a',
+                UID: '4934ee07118197xx3f74d5xxxx7b0076',
+                time: '2024-07-12T14:45:25+0000',
+                action: 'browse',
+                bmID: '',
+                first: 0,
+                properties: {
+                  category: 'Shirts',
+                  url: 'http://example.com/shirts',
+                  description: 'A really cool khaki shirt.',
+                  price: 9.99,
+                  title: 'Khaki Shirt',
+                  test_key: 'value',
+                },
+              },
+            },
+            source: {},
+          },
+        ],
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+      },
+      pathSuffix: '',
+    },
+    output: {
+      response: {
+        status: 200,
+        body: [
+          {
+            output: {
+              batch: [
+                {
+                  context: {
+                    library: {
+                      name: 'unknown',
+                      version: 'unknown',
+                    },
+                    integration: {
+                      name: 'Cordial',
+                    },
+                    traits: {
+                      userId: '6690fe3655e334xx028xxx',
+                      email: 'jondoe@example.com',
+                      _id: '6690fe3655e334xx028xxx',
+                      channels: {
+                        email: {
+                          address: 'jondoe@example.com',
+                          subscribeStatus: 'subscribed',
+                          subscribedAt: '2024-07-12T09:58:14+0000',
+                        },
+                      },
+                      createdAt: '2024-07-12T09:58:14+0000',
+                      address: {
+                        city: 'San Miego',
+                      },
+                      first_name: 'John',
+                      last_name: 'Doe',
+                      lastUpdateSource: 'api',
+                      lastModified: '2024-07-12T13:00:49+0000',
+                      cID: '6690fe3655e334xx028xxx',
+                    },
+                    device: {
+                      type: 'computer',
+                      device: false,
+                      platform: false,
+                      browser: false,
+                      robot: true,
+                    },
+                    externalId: [],
```

Comment:
This should not come as empty array. since we already have cID available in contact.

Reply:
done